Ghidra/Processors/V850E2M/data/languages/Helpers/Conditions.sinc

```diff
@@ -0,0 +1,38 @@
+#####################################################
+#####		     Conditions			#####
+#####################################################
+
+
+c0003: "V"	is op0003=0x0	{ tmp:1 = ($(OV)) == 1; 		export tmp; }
+c0003: "NV"	is op0003=0x8	{ tmp:1 = ($(OV)) == 0; 		export tmp; }
+c0003: "C_L"	is op0003=0x1	{ tmp:1 = ($(CY)) == 1; 		export tmp; }
```

Comment:
please rewrite all names of conditions in lower case, so that they match the rest of the code.